OriginateTheme/Scripts/ot_generator.py

```diff
@@ -449,11 +518,12 @@ def generateComponentsClassHeaderFileContent(className, components):
             Object containing all key/value pairs for the component definitions.
     """
     # Component specific lambda functions.
-    createComponentPropertyName = lambda componentKey, key, uiType: componentKey + upcaseFirstLetter(key) + upcaseFirstLetter(uiType)
+    createComponentPropertyName = lambda componentKey, key, uiType:  componentKey + upcaseFirstLetter(key) + upcaseFirstLetter(uiType)
     createPropertyDefinition = lambda componentKey, key, uiType, accessType: createProperty('strong', accessType, 'UI' + upcaseFirstLetter(uiType), createComponentPropertyName(componentKey, key, uiType))
+    createValuePropertyDefinition = lambda componentKey, key, objcType, accessType: createValueProperty(accessType, objcType, createComponentPropertyName(componentKey, key, ""))
 
     # Create the public properties.
-    OriginateThemePublicProperties = [[[createPropertyDefinition(component.key, c.key, 'color', 'readonly') for c in component.colors], [createPropertyDefinition(component.key, f.key, 'font', 'readonly') for f in component.fonts], ['']] for component in components]
+    OriginateThemePublicProperties = [[[createPropertyDefinition(component.key, c.key, 'color', 'readonly') for c in component.colors], [createPropertyDefinition(component.key, f.key, 'font', 'readonly') for f in component.fonts], [createValuePropertyDefinition(component.key, b.key, 'BOOL', 'readonly') for b in component.bools], ['']] for component in components]
```

Comment:
can we put each element of this array on a newline and align them? Maybe something like this?

```python
OriginateThemePublicProperties = [[
  [createPropertyDefinition(component.key, c.key, 'color', 'readonly') for c in component.colors],
  [createPropertyDefinition(component.key, f.key, 'font', 'readonly') for f in component.fonts],
  [createValuePropertyDefinition(component.key, b.key, 'BOOL', 'readonly') for b in component.bools],
  ['']
] for component in components]
```